### DISCUSSION TITLE
Some news about .NET 10: BackgroundServices and new String Comparer

### DISCUSSION THREAD

**Discussion by giscus** [linkdotnet/Blog.Discussions#127] [General]
# Some news about .NET 10: BackgroundServices and new String Comparer

Recently, there were two new features merged in .NET 10 I think are small little quality of life improvements: Avoid Blocking on startup with BackgroundServices and a new string comparer.

https://steven-giesel.com/blogPost/3a8c29a3-750a-40f0-aa43-c236b855813e

**Comment by kapsiR** [comment]
From my understanding, the API change will [**not happen**](https://github.com/dotnet/runtime/issues/36063#issuecomment-2515480657) (despite the breaking change of course):
> IOW, we like the behavior breaking change but not the API.
>
> We will close this issue (as it tracks the API). There will be another issue.

**Reply by linkdotnet** [reply]
Oh fair point - will correct that.
